### PR TITLE
permalink for todo page which filter data by type

### LINKED ISSUE #675: Create permalink for filters in to-do page
**Describe the solution you'd like**

I expect a permalink which points to a selected filter in to-do page, something like:
https://developers.italia.it/it/cosa-fare?type=hacktoberfest

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by sebbalex**
## Description

As already mentioned in #675 , this PR tackles:
* This will handle permalink which will filters data by type
* inputs are sanitized

## Checklist

- [X] I followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md)
- [X] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [X] Have you written new tests for your core changes, as applicable?
- [X] Have you successfully ran tests with your changes locally?
- [X] Ready for review! :rocket:

## Fixes

- Fixes #675

### CODE REVIEW

File: assets/js/github-issues.js [hunk 1]
```diff
@@ -133,12 +137,23 @@ $(document).ready(function () {
...
 
+            // Get type from query param (useful to share links)
+            const urlParams = new URLSearchParams(window.location.search);
+            const typeParam = encodeHTML(urlParams.get('type'));
+            let out = [];
+            $github_types_list.find('input').each((n, i) => {
+                out.push(i.value);
+              });
+            if(typeParam && out.includes(typeParam)) {
+                $check = $('#github_types_' + (out.indexOf(typeParam) +1 ));
```
[the remark this answers is not in the document]
Comment (sebbalex) [reply assets/js/github-issues.js]: :+1:

File: assets/js/github-issues.js [hunk 2]
```diff
@@ -133,12 +137,23 @@ $(document).ready(function () {
...
                     }
                 });
-
             });
 
+            // Get type from query param (useful to share links)
+            const urlParams = new URLSearchParams(window.location.search);
+            const typeParam = encodeHTML(urlParams.get('type'));
+            let out = [];
+            $github_types_list.find('input').each((n, i) => {
```
[the remark this answers is not in the document]
Comment (sebbalex) [reply assets/js/github-issues.js]: nice catch :)

File: assets/js/github-issues.js [hunk 3]
```diff
@@ -25,6 +25,10 @@ $(document).ready(function () {
               </div>`;
     }
 
+    function encodeHTML(s) {
+        return s.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+    }
+
```
[the remark this answers is not in the document]
Comment (sebbalex) [reply assets/js/github-issues.js]: :+1:

**Review by bfabio — APPROVED** [review]
:rocket:

**Pull request merged**